source/vibe/http/fileserver.d

```diff
@@ -162,7 +160,32 @@ class HTTPFileServerSettings {
 		this();
 		serverPathPrefix = path_prefix;
 	}
-} 
+
+	deprecated("Use .options and HTTPFileServerOption.failIfNotFound instead.")
+	@property bool failIfNotFound() const { return options & HTTPFileServerOption.failIfNotFound; }
+
+	deprecated("Use .options and HTTPFileServerOption.failIfNotFound instead.")
+	@property void failIfNotFound(bool val) {
+		if (val)
```

Comment:
options = (options & ~HTTPFileServerOption.failIfNotFound) | (-cast(int)val & HTTPFileServerOption.failIfNotFound);
Without ifs it's shorter and may work.